### ISSUE TITLE
NTR: Skin biopsy: kindlin-1 antibody staining abnormal or absent by immunofluorescent antibody/antigen mapping (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9173]
New term request Skin biopsy: kindlin-1 antibody staining abnormal or absent by immunofluorescent antibody/antigen mapping (elthat_100813153212):

**Comment by pnrobinson** [comment]
@MickeySegal
just to confirm that we mean this protein?:

FERM domain containing kindlin 1
https://www.ncbi.nlm.nih.gov/gene/55612

**Comment by MickeySegal** [comment]
The FERM domain part of the terminology is used to describe the gene, but it seems not to be mentioned when describing the antibody: https://www.ncbi.nlm.nih.gov/books/NBK349072/  "Although anti-kindlin-1 immunostaining is also applied as a diagnostic test for KS, the final confirmation is made by FERMT1 molecular genetic testing"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000714
- label: Reduced epidermal kindlin-1 expression
- parents: [Abnormal epidermal protein expression (HP:0034573)](https://hpo.jax.org/app/browse/term/HP:0034573)
- pmid: PMID:14962093
- def: Lower than normal amount of kindlin-1 in the epidermis.
- comment: In control skin, immunofluorescence labeling with antikindlin-1 antibody shows bright, pan-epidermal staining. In individuals with Kindler syndrome (OMIM 173650), staining is markedly reduced and in some cases absent.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9173
- synonyms:
